### ISSUE TITLE
"Add triggered breakpoint" should move focus into breakpoint widget

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#203154]
Testing #202882

1. F1 > add triggered breakpoint
2. Breakpoint widget opens
3. Focus stays in editor 🐛 

The focus should be moved to the Breakpoint widget for smoother flow and accessibility.
For example, F1 > add conditional breakpoint moves the focus
